src/main/java/com/google/cloud/genomics/dataflow/readers/VariantStreamer.java

```diff
@@ -79,7 +81,7 @@ public static StreamVariantsRequest getVariantRequests(String variantSetId) {
         .transformAndConcat(new Function<Contig, Iterable<Contig>>() {
             @Override
             public Iterable<Contig> apply(Contig contig) {
-              return contig.getShards(100000000L);
+              return contig.getShards(5000000L);
```

Comment:
+1 on moving it up to a constant. Add a TODO that it should be replaced
with a better heuristic

On Wed, Jul 1, 2015 at 10:23 AM, Paul Grosu <notifications@github.com>
wrote:

> In
> src/main/java/com/google/cloud/genomics/dataflow/readers/VariantStreamer.java
> <https://github.com/googlegenomics/dataflow-java/pull/106#discussion_r33701691>
> :
>
> > @@ -79,7 +81,7 @@ public static StreamVariantsRequest getVariantRequests(String variantSetId) {
> >          .transformAndConcat(new Function<Contig, Iterable<Contig>>() {
> >              @Override
> >              public Iterable<Contig> apply(Contig contig) {
> > -              return contig.getShards(100000000L);
> > +              return contig.getShards(5000000L);
>
> See my previous comment regarding hardcoding it:
>
> https://github.com/googlegenomics/dataflow-java/pull/106/files#r33701593
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/dataflow-java/pull/106/files#r33701691>
> .
>